app/javascript/app/service-provider-form.js

```diff
@@ -107,51 +177,13 @@ $(function () {
   if (!$('.service-provider-form').length) {
     return;
   }
-
-  // Selectors
-  const fileContainer = $('#certificate-container');
-  const logoInput = $('.input-file');
-  const pemInputMessage = $('.js-pem-input-error-message');
-  const pemInput = $('.js-pem-input');
   const redirectURI = $('#add-redirect-uri-field');
 
   // Functions
-
-  const setPemError = (message) => (pemInputMessage[0].textContent = message); // eslint-disable-line
-
   redirectURI.click(() =>
       $('.service_provider_redirect_uris input:last-child')
           .clone()
           .val('')
           .appendTo('.service_provider_redirect_uris')
   );
```

Comment:
Was this meant to be removed (replaced by `redirectURISetup`) ?

Reply:
Yes, whoops